doc/guides/maintaining-V8.md

```diff
@@ -0,0 +1,281 @@
+
+*   V8 only supports active branches. There is no testing done on any branches older than the current stable/beta/master.
+*   A fix needing backport is tagged w/ *merge-request-x.x* tag. This can be done by anyone interested in getting the fix backported. Issues with this tag are reviewed by the V8 team regularly as candidates for backporting.
+*   Fixes need some 'baking time' before they can be approved for backporting. This means waiting a few days to ensure that no issues are detected on the canary/beta builds.
+*   Once ready, the issue is tagged w/ *merge-approved-x.x* and one can do the actual merge by using the scripts on the [wiki page](https://github.com/v8/v8/wiki/Merging%20&%20Patching).
+*   Merge requests to an abandoned branch will be rejected.
+*   Only bug fixes are accepted for backporting.
+
+# Node Support Requirements
+
+At any given time Node needs to be maintaining a few different V8 branches for the various Current, LTS, and nightly releases. At present this list includes the following branches<sup>1</sup>:
+
+<table>
+  <tr>
+   <td><strong>Release</strong>
+   </td>
+   <td><strong>Support Start</strong>
+   </td>
+   <td><strong>Support End</strong>
+   </td>
+   <td><strong>V8 version</strong>
+   </td>
+   <td><strong>V8 branch released</strong>
+   </td>
+   <td><strong>V8 branch abandoned</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Node v4.x
+   </td>
+   <td>2015-10-01
+   </td>
+   <td>2018-04-01
+   </td>
+   <td>4.5
+   </td>
+   <td>2015-09-01
+   </td>
+   <td>2015-10-13
+   </td>
+  </tr>
+  <tr>
+   <td>Node v6.x
+   </td>
+   <td>2016-04-01
+   </td>
+   <td>2019-04-01
+   </td>
+   <td>5.1
+   </td>
+   <td>2016-05-31
+   </td>
+   <td>2016-06-26
+   </td>
+  </tr>
+  <tr>
+   <td>Node v7.x
+   </td>
+   <td>2016-10-01
+   </td>
+   <td>2017-04-01
+   </td>
+   <td>5.4
+   </td>
+   <td>2016-10-18
+   </td>
+   <td>~2016-12-01
+   </td>
+  </tr>
+  <tr>
+   <td>master
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>5.4
+   </td>
+   <td>2016-10-18
+   </td>
+   <td>~2016-12-01
+   </td>
+  </tr>
+</table>
+
+
+The versions of V8 used in Node v4.x and v6.x have already been abandoned by upstream V8. However, Node.js needs to continue supporting these branches for many months (Current branches) or several years (LTS branches).
+
+# Maintenance Process
+
+Once a bug in Node.js has been identified to be caused by V8, the first step is to identify the versions of Node and V8 affected. The bug may be present in multiple different locations, each of which follows a slightly different process.
+
+* Unfixed bugs. If the bug exists in the V8 master branch.
+* Fixed, but needs backport. The bug may need porting to one or more branches.
+    * Backporting to active branches.
+    * Backporting to abandoned branches.
+* Backports identified by the V8 team.
```

Comment:
is this different from backporting to active branches? doesn't v8 backport to active?